src/items.rs

```diff
@@ -2501,14 +2507,23 @@ fn rewrite_where_clause_rfc_style(
         span_end,
         false,
     );
-    let comma_tactic = if where_clause_option.suppress_comma {
-        SeparatorTactic::Never
+    let comma_tactic =
+        if where_clause_option.suppress_comma || context.config.where_single_line() && len == 1 {
```

Comment:
Could you factor out `context.config.where_single_line() && len == 1` into a variable?